pom.xml

```diff
@@ -287,6 +287,11 @@
 				<enabled>false</enabled>
 			</releases>
 		</repository>
+		<!--Captcha wrapper repository-->
```

Comment:
Please remove mvn repo for reCaptcha, since it's now part of the code.

Reply:
Repo removed.